Octokit/Clients/MiscellaneousClient.cs

```diff
@@ -82,7 +82,38 @@ public MiscellaneousClient(IConnection connection)
             var endpoint = new Uri("gitignore/templates/" + Uri.EscapeUriString(templateName), UriKind.Relative);
 
             var response = await _connection.Get<GitIgnoreTemplate>(endpoint, null, null)
-                                  .ConfigureAwait(false);
+                .ConfigureAwait(false);
```

Comment:
I'm indifferent on which way is correct, but either this one is right or the new code below is right.